projects/gnomad/src/server/server.js

```diff
@@ -29,8 +38,56 @@ if (JSON.parse(process.env.ENABLE_HTTPS_REDIRECT || 'false')) {
   })
 }
 
+//var myLogger = function (req, res, next) {
+
+//  var wholeCookie = req.cookies
+//  var chorus_accessToken = req.cookies['accessToken']
+//  console.log(chorus_accessToken)
+
+//  var jwtDecode = require('jwt-decode')
+//  let decoded = jwtDecode(chorus_accessToken)
+//  var current_time = (new Date().getTime() / 1000)
+  
+//  if (current_time > decoded.exp){
+//    console.log('EXPIRED')
+//    res.redirect(`127.0.0.1/auth`)  
+//  }
+//  next()
+//}
+//app.use(myLogger)
+
+app.use((request, response, next) => {
+  // get cookies
+  var isRedirect = true
+  var jwtDecode = require('jwt-decode')
+  var cookieVal = request.cookies['accessToken']
+  console.log('Cookie check', 'YES')
+
+  if (typeof cookieVal !== 'undefined' && cookieVal !== null ) {
+    let decoded = jwtDecode(cookieVal)
+
+    if ((new Date().getTime() / 1000) < decoded.exp){ //check if not expired
+      console.log('isVALID: ', true)
+      isRedirect = false
+    }
+  }
+
+  if (isRedirect === true) {
+    console.log('isVALID: ', false)
+    response.clearCookie('accessToken', {path: '/'})
+    response.redirect(302,'http://46.137.240.42/auth')
```

Comment:
```suggestion
    response.redirect(302,'http://localhost/auth')
```
the IP should not be hard codded... does localhost works ?

Reply:
Updated in next commit